scripts/mulmoclaude/deps.mjs

```
@@ -0,0 +1,214 @@
+// code. The `[\s\S]*?` lazy match lets one pattern cover every
+// brace-y variant without listing each shape separately.
+const IMPORT_PATTERNS = [
+  // Single-line forms without braces:
+  //   import X from "pkg"
+  //   import * as Y from "pkg"
+  //   export * from "pkg"
+  //   export type T from "pkg"
+  // `[^{\n]*` keeps the match on one line and bails before any
+  // brace block, so free-form "from" tokens elsewhere in the file
+  // can't be absorbed into the match.
+  /^\s*(?:import|export)\b[^{\n]*\sfrom\s+['"]([^'"]+)['"]/gm,
```

ESLint

[error] Make sure the regex used here, which is vulnerable to super-linear runtime due to backtracking, cannot lead to denial of service.

(sonarjs/slow-regex)